### PR TITLE
Minor text cleanup

### PR DESCRIPTION
**Pull request by calebjohn24**
#### Pull Request checklist

- [] Addresses an issue: Fixes #0000

#### Description of changes

(Summarize any changes made)

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -54,11 +54,11 @@ const Home: NextPage = () => {
         verticalAlignment="center"
       >
         <LogoIcon color="white" size="display" />
-        <Display color="white" align="center">
-          Turn Surveys Into Sales
-        </Display>
+        <Title color="white" align="center">
```
Comment (czearing) [inline src/pages/index.tsx]: I would keep this as Display instead of Title

**Review by czearing — APPROVED** [review]
This looks good. I would revert the Title change though as Display is easier to read

File: src/pages/index.tsx (hunk 1 above)
Comment (calebjohn24) [reply src/pages/index.tsx]: This is way too big on mobile thats why I changed

Comment (calebjohn24) [reply src/pages/index.tsx]: alright fixed

**Pull request merged**